teamscale-maven-plugin/src/main/java/com/teamscale/tia/maven/TiaCoverageConvertMojo.java

```diff
@@ -0,0 +1,135 @@
+package com.teamscale.tia.maven;
+
+import com.google.common.base.Strings;
+import com.teamscale.jacoco.agent.options.AgentOptionParseException;
+import com.teamscale.jacoco.agent.options.ClasspathUtils;
+import com.teamscale.jacoco.agent.options.FilePatternResolver;
+import com.teamscale.jacoco.agent.util.Benchmark;
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.plugins.annotations.ResolutionScope;
+import shadow.com.teamscale.client.TestDetails;
+import shadow.com.teamscale.report.EDuplicateClassFileBehavior;
+import shadow.com.teamscale.report.ReportUtils;
+import shadow.com.teamscale.report.testwise.ETestArtifactFormat;
+import shadow.com.teamscale.report.testwise.TestwiseCoverageReportWriter;
+import shadow.com.teamscale.report.testwise.jacoco.JaCoCoTestwiseReportGenerator;
+import shadow.com.teamscale.report.testwise.model.TestExecution;
+import shadow.com.teamscale.report.testwise.model.factory.TestInfoFactory;
+import shadow.com.teamscale.report.util.ClasspathWildcardIncludeFilter;
+import shadow.com.teamscale.report.util.CommandLineLogger;
+import shadow.com.teamscale.report.util.ILogger;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Batch converts all created .exec file reports into a testwise coverage report.
+ */
+@Mojo(name = "testwise-coverage-converter", defaultPhase = LifecyclePhase.VERIFY,
+		requiresDependencyResolution = ResolutionScope.RUNTIME, threadSafe = true)
```

Comment:
Do you know what `requiresDependencyResolution = ResolutionScope.RUNTIME` is used for?

Reply:
I guess this only tells maven that the dependencies do not have to be resolved during compilation, but at runtime. I think it does not make a great difference, but I adapted this from our other plugin mojos.